packages/qdrant-loader/tests/unit/core/worker/test_pool.py

```
@@ -308,6 +308,67 @@ async def long_handler(_type: str, _payload: dict) -> None:
+    handler_started = asyncio.Event()
+    handler_completed = asyncio.Event()
+
+    async def long_handler(_job_type: str, _payload: dict) -> None:
+        handler_started.set()
+        await asyncio.sleep(10)  # Would block for 10s without cancellation
+        handler_completed.set()  # Must NOT be reached
+
+    original_extend = sqlite_job_queue.extend_visibility
+
+    async def claim_lost_extend(
+        _job_id: int, _lease_seconds: int, claim_attempt: int
```

Ruff (0.15.17)

[warning] Unused function argument: `claim_attempt`

(ARG001)